### PR TITLE
Release mkr plugin install

### PR DESCRIPTION
**Pull request by shibayu36**

### CODE REVIEW

File: plugin/commands.go [hunk 1]
```diff
@@ -23,28 +23,53 @@ import (
...
+    - <owner>/<repo>[@<release_tag>]
+          Install from specified github owner, repository, and Github Releases tag.
+          If you omit <release_tag>, the installer install from latest Github Release.
+          Example: mkr plugin install mackerelio/mackerel-plugin-sample@v0.0.1
+    - <plugin_name>[@<release_tag]
+          Install from plugin registry.
+          You can find available plugins in https://github.com/mackerelio/plugin-registry
+          Example: mkr plugin install mackerel-plugin-sample
+
+    The installer uses Github API to find latest release.  Please set a github token to
```
Comment (itchyny) [inline plugin/commands.go]: [nit] `the latest release`

File: plugin/commands.go [hunk 2]
```diff
@@ -23,28 +23,53 @@ import (
...
+    GITHUB_TOKEN environment variable, or to github.token in .gitconfig.
+    Otherwise, installing sometimes fails because of Github API Rate Limit.
```
Comment (itchyny) [inline plugin/commands.go]: [nit] installation sometimes fails ...

**Comment by itchyny** [comment]
LGTM!

**Comment by shibayu36** [comment]
I fix command manual by https://github.com/mackerelio/mkr/pull/125/commits/6e5d4abca8800f8e528ed156d8cf7902386826c7 .

**Comment by shibayu36** [comment]
I'll merge after a test is passed.

**Pull request merged**